src/web/mjs/connectors/NetEaseComic.mjs

```diff
@@ -6,6 +6,7 @@ export default class NetEaseComic extends Connector {
     constructor() {
         super();
         super.id = 'neteasecomic';
+        // TODO: rename website?
```

Comment:
I would say yes if existing bookmarks don't break.

Reply:
Has been renamed. Only ID is critical because it will break all existing bookmarks, so it should not be changed.